library/build.gradle

```diff
@@ -37,6 +24,8 @@ sourceSets {
     }
 }
 
+archivesBaseName = 'PokeGOAPI-library'
```

Comment:
One way to allow more control from the master build script would be `archivesBaseName = archivesBaseName + '-library'` where the master build script has `archivesBaseName = 'PokeGOAPI'`